qiskit/test/mock/fake_backend_v2.py

```diff
@@ -176,11 +180,6 @@ def _default_options(cls):
     def run(self, run_input, **options):
         raise NotImplementedError
 
-    def qubit_properties(self, qubit):
-        if isinstance(qubit, int):
-            return self._qubit_properties[qubit]
-        return [self._qubit_properties[i] for i in qubit]
-
 
 class FakeBackendSimple(BackendV2):
     """A fake simple backend that wraps BasicAer to implement run()."""
```

Comment:
This was not introduced by this PR but I think it would be good to explain in the docstring the difference between `FakeBackendV2` and `FakeBackendSimple`.